src/main/java/com/epam/test_generator/entities/Case.java

```diff
@@ -39,29 +34,31 @@
     private Set<Tag> tags;
 
     public Case() {
-        SimpleDateFormat formatter = new SimpleDateFormat("MM/dd/yyyy");
+        final SimpleDateFormat formatter = new SimpleDateFormat("MM/dd/yyyy");
         creationDate = formatter.format(Calendar.getInstance().getTime());
```

Comment:
I'd recommend excluding time to a local variable because if GC is initiated between getTime executions and you'll get different values in creationDate and updateDate.

Reply:
Now, this part of the functionality is implemented in a different way. So this local variable is absent for now.